src/components/services/views.tsx

```diff
@@ -166,3 +202,60 @@ export function ServicePage(props: IServicePageProperties): ReactElement {
     </ServiceTab>
   );
 }
+
+function FileListingItem(props: IFileListingItemProperties): ReactElement {
+  return <li className="service-log-list-item">
+    <a className="govuk-link" download href={props.link} aria-describedby={`download-${props.date.getTime()}`}>
+      {props.name}
+    </a>
+
+    <p className="govuk-body" id={`download-${props.date.getTime()}`}>
+      <span className="govuk-visually-hidden">file type </span>
+      <span className="service-log-list-item__attribute"><abbr title="record of events">LOG</abbr></span>,
+      {' '}
+      <span className="govuk-visually-hidden">file size </span>
+      <span className="service-log-list-item__attribute">{bytesToHuman(props.size)}</span>
+      <span className="service-log-list-item__attribute">Last written: {moment(props.date).format(DATE_TIME)}</span>
+    </p>
+  </li>;
+}
+
+export function ServiceLogsPage(props: IServiceLogsPageProperties): ReactElement {
+  return <>
+  <ServiceTab {...props}>
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <p className="govuk-body">
+            The following are hourly chunks of downloadable logs for this database.
+          </p>
+          <p className="govuk-body">
+            At the moment, we only provide up to 72 hours of logs. Get in touch if you need a larger range of data.
+          </p>
+          <p className="govuk-body">
+            Log timestamps are in UTC format.
+          </p>
+
+          <hr className="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
+
+          {props.files.length > 0
+            ? <ul className="govuk-list service-log-list">
+                {props.files.map(file => <FileListingItem
+                  date={new Date(file.LastWritten)}
+                  key={file.LogFileName}
+                  link={props.linkTo('admin.organizations.spaces.services.logs.download', {
+                    filename: file.LogFileName,
+                    organizationGUID: props.organizationGUID,
+                    serviceGUID: props.service.metadata.guid,
+                    spaceGUID: props.spaceGUID,
+                  })}
+                  name={file.LogFileName}
+                  size={file.Size}
+                />)}
+              </ul>
+            : <p className="govuk-body">There are no log files available at this time. Please comeback later.</p>
```

Comment:
```suggestion
            : <p className="govuk-body">There are no log files available at this time. Please come back later.</p>
```

Comment above about wording

Reply:
Deleted.